CONTRIBUTING.md

```diff
@@ -161,4 +167,5 @@ This library uses [semantic versioning](http://semver.org/). For each release, t
         - Incomplete Javadoc
 8. Add a tag and push to origin - e.g.: `git tag v1.2.4 && git push origin v1.2.4`
 9. Create the release on Github including populating the release notes
-10. Create the entry on the [Ably Changelog](https://changelog.ably.com/) (via [headwayapp](https://headwayapp.co/))
+10. Merge any [website docs](https://github.com/ably/docs) PRs related to your changes. At minimum, you must update the setup/installation section to point to the latest SDK version.
```

Comment:
```suggestion
10. Create and merge PR to update [website docs](https://github.com/ably/docs) related to your changes. At minimum, you must update the setup/installation section to point to the latest SDK version.
```

Reply:
`Merge any` sounds like you have to merge existing PR, but we need to create explicit PR for related changes and then merge it.

Reply:
I would suggest keeping the original version - we don't want to suggest that you're thinking about writing docs just as you're cutting the release!

Reply:
Then step should be done before cutting the release, wdyt

Reply:
Because, it's mandatory to create a PR to update docs ( minimum version update PR is needed )

Reply:
4984a07

Reply:
LGTM